common/persistence/nosql/nosql_execution_store.go

```diff
@@ -830,13 +830,14 @@ func (d *nosqlExecutionStore) CreateFailoverMarkerTasks(
 ) error {
 
 	var nosqlTasks []*nosqlplugin.ReplicationTask
-	for _, task := range request.Markers {
+	for i, task := range request.Markers {
 		ts := []persistence.Task{task}
 
 		tasks, err := d.prepareReplicationTasksForWorkflowTxn(task.DomainID, rowTypeReplicationWorkflowID, rowTypeReplicationRunID, ts)
 		if err != nil {
 			return err
 		}
+		tasks[i].CurrentTimeStamp = request.CurrentTimeStamp
```

Comment:
It is redundant here to add timestamp to each task since we are storing the entire tasklist into Cassandra; also we do have a creationTime for each task. 

I am thinking how to pass one timestamp into the method `d.db.InsertReplicationTask` without changing the signature. @taylanisikdemir Any suggestions? I am thinking to add the timestamp into ctx, but not sure if it is the best way.

Reply:
There has been update in this block of code (i.e. Refactor peristence layer to support inserting history tasks of new categories #6671). Adjusted the code accordingly.

Reply:
There have been another new changes and this block of code has been restored.